app/src/main/res/layout/view_user_profile.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:paddingTop="@dimen/profile_padding_vertical_default"
+    android:paddingStart="@dimen/profile_padding_horizontal_default"
+    android:paddingEnd="@dimen/profile_padding_horizontal_default"
+    android:paddingBottom="@dimen/profile_padding_vertical_default"
+    android:dividerPadding="@dimen/profile_padding_divider" >
+
+    <Spinner
+        android:id="@+id/sex_spinner"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:hint="@string/profile_sex_prompt"/>
+
+    <EditText
+        android:id="@+id/birthday_edit_text"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:editable="false"
+        android:hint="@string/profile_birthday_prompt"/>
```

Comment:
If you focus on another text field and then refocus this one, it opens a keyboard and not the calendar picker.